components/admin_console/admin_definition.jsx

```diff
@@ -4062,10 +4062,14 @@ const AdminDefinition = {
                         defaultMessage='deprecated'
                     />
                 ),
-                shouldDisplay: () => false,
+                shouldDisplay: (license) => license.IsLicensed && license.OpenId === 'true',
             },
             isHidden: it.any(
                 it.not(it.licensed),
+                it.all(
+                    it.licensedForFeature('OpenId'),
+                    it.not(usesLegacyOauth),
+                ),
```

Comment:
OAuth should be hidden if it is licenced for OpenId AND does NOT use legacy oauth.